tests/unit/hmrc/mtdFraudPrevention.test.js

```diff
@@ -58,6 +58,8 @@ describe("FraudPreventionHeaders", () => {
     );
     expect(headers.get("Gov-Client-Browser-Do-Not-Track")).toBe("true");
     expect(headers.get("Gov-Client-Local-IPs")).toBe("127.0.0.1,127.0.0.2");
+    expect(headers.get("Gov-Client-Device-ID").split("-").length).toEqual(5);
+    expect(headers.get("Gov-Client-Device-ID").length).toEqual(36);
```

Comment:
I would recommend making the same change here as in the other test. Mock uuid and verify for an exact match.